ngmaster/1.0.0/Dockerfile

```diff
@@ -55,10 +52,24 @@ WORKDIR /data
 FROM app as test
 
 # so that mamba/conda env is active when running below commands
-ENV ENV_NAME="ngmaster-env"
+ENV ENV_NAME="base"
 ARG MAMBA_DOCKERFILE_ACTIVATE=1
 
 # show help and version outputs; run the program's internal tests
 RUN ngmaster --help && echo && \
  ngmaster --version && mlst --version && echo && \
  ngmaster --test
+
+# getting unzip for unziping archive downloaded from NCBI
+RUN apt-get update && apt-get install unzip curl ca-certificates -y --no-install-recommends
+
+# so that testing outputs are in /test
+WORKDIR /test
+
+# test with an actual assembly downloaded from RefSeq
+# more info on this genome here: https://www.ncbi.nlm.nih.gov/labs/data-hub/genome/GCF_013030075.1/
+RUN curl -OJX GET "https://api.ncbi.nlm.nih.gov/datasets/v2alpha/genome/accession/GCF_013030075.1/download?include_annotation_type=GENOME_FASTA,GENOME_GFF,RNA_FASTA,CDS_FASTA,PROT_FASTA,SEQUENCE_REPORT&filename=GCF_013030075.1.zip" \
+ -H "Accept: application/zip" && \
+ unzip GCF_013030075.1.zip && \
+ ngmaster /test/ncbi_dataset/data/GCF_013030075.1/GCF_013030075.1_ASM1303007v1_genomic.fna > ngmaster.out.tsv && \
+ cat /test/ngmaster.out.tsv
```

Comment:
I don't have a real problem with this - so it's more of a comment for me, but I feel like

```
RUN wget https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/013/030/075/GCA_013030075.1_ASM1303007v1/GCA_013030075.1_ASM1303007v1_genomic.fna.gz && gzip -d GCA_013030075.1_ASM1303007v1_genomic.fna.gz
```

takes up fewer lines.

Reply:
haha yeah, I copied the command off the NCBI datasets page. Happy to adjust it though

Reply:
I don't think you need to. I wonder by NCBI gives such a long command, though.

Reply:
OK, I'll leave as is. I've rebuilt the image locally and the test layer runs successfully, so I think this is ready to merge